packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -195,19 +197,48 @@ export async function createNewFluidFileFromSummary(
             logger,
             { eventName: "createNewFile" },
             async (event) => {
-                const { url, headers } = getUrlAndHeadersWithAuth(
-                    initialUrl, storageToken, forceAccessTokenViaAuthorizationHeader);
-                headers["Content-Type"] = "application/json";
+                const snapshotBody = JSON.stringify(containerSnapshot);
```

Comment:
Should UMP call be conditional on size of payload? What if it's larger than 80K?

Reply:
I do have a size check below it :)